pre-build-events-cpp.bat

```diff
@@ -0,0 +1,27 @@
+REM This script is invoked before compiling an assembly, and if the target file exist, it moves it to a temporary location
```

Comment:
Called in pre-build events to clear out files that are locked when the corprofiler reaches into dlls controlled by iisexpress.exe

Reply:
Moving into another PR.